packages/manager/modules/pci/src/projects/new/payment/controller.js

```diff
@@ -67,12 +68,12 @@ export default class PciProjectNewPaymentCtrl {
   pollCheckDefaultPaymentMethod(paymentMethodId, currentTime = 0) {
     return this.$timeout(() => {
       return this.getPaymentMethod(paymentMethodId).then((paymentMethod) => {
-        if (paymentMethod.isValid()) {
+        if (paymentMethod.status === 'VALID') {
```

Comment:
As seen together, ok for the hotfix, but the better would be to abstract this test, and fix the `paymentMethod.isValid` method